app/models/types_de_champ/type_de_champ_base.rb

```diff
@@ -1,11 +1,12 @@
 class TypesDeChamp::TypeDeChampBase
   include ActiveModel::Validations
 
-  delegate :description, :libelle, :mandatory, :stable_id, to: :@type_de_champ
+  delegate :description, :libelle, :mandatory, :stable_id, :fillable?, to: :@type_de_champ
 
-  FILL_DURATION_SHORT  = 10.seconds.in_seconds
+  FILL_DURATION_SHORT  = 10.seconds
   FILL_DURATION_MEDIUM = 1.minute.in_seconds
   FILL_DURATION_LONG   = 3.minutes.in_seconds
```

Comment:
`10.seconds.class.name` => `Duration`
`1.minute.in_seconds.class.name` => `Integer` 

Est ce qu'on passerait pas tout en `Duration` ?

```suggestion
  FILL_DURATION_MEDIUM = 1.minute
  FILL_DURATION_LONG   = 3.minutes
```